docs/source/substrafl_doc/substrafl_overview.rst

```diff
@@ -165,3 +168,23 @@ This also means that for the final round of the strategy, we do a useless step o
 - Reuse the same function as the other train tasks, which speeds up the execution
 
 For a more detailed example, see the :ref:`Federated Averaging <substrafl_doc/api/algorithms:TorchFedAvgAlgo>` implementation.
+
+Local and shared state
+^^^^^^^^^^^^^^^^^^^^^^
+
+SubstraFL creates :ref:`tasks <concept_task>` for the user in order to ease the implementation of Federated :ref:`Compute Plans <concept_compute_plan>`.
+
+The concept of local and shared state is used in SubstraFL in order to differentiate which information we want to keep locally on an organization, and which information
+we want to share with the other organization.
+
+Local state
+***********
+
+A local state is a task output or input that goes from a task within an organization to the next task that will be executed on that organization. A local state
+is useful to keep internal states along a Federated Compute Plan within an organization. A local state may be, for instance, a dictionary to be loaded in order to
+re-initialize objects and variable to their previous values.
```

Comment:
Could you give a ML example? Like the optimize state? the model architecture? etc